tests/tests/test_github_webhooks.py

```diff
@@ -0,0 +1,605 @@
+        "debug:Trying to @mention the user in the newly created PR",
+        "debug:userName: tranchitella",
+        'github.CreateComment: org=mendersoftware,repo=workflows-enterprise,number=0,comment={"body":"@tranchitella I have created a PR for you, ready to merge as soon as tests are passed"}',
+        "info:Pull request event with action: closed",
+        "info:workflows:140 would trigger 0 builds",
+    ]
```

Comment:
This one doesn't make sense. The payload is a PR with `"action": "closed"`, so it should not do any of this (updating the pr_branch, starting the pipeline, etc). Maybe our mocking is making the code misbehave here?

Reply:
I don't think so, looking at the code, this is what we do on action: closed.
Am I wrong?

Reply:
Found the issue: the test fixture was scope=session, and this made the logs to be retained across multiple tests, appending to the same list. It is now fixed.